pkg/core/session.go

```diff
@@ -27,6 +27,7 @@ var (
 	ErrPropertiesCallFailed        = errors.New("the properties call returned non-zero")
```

Comment:
@yogev-lb can you elaborate in commit message why you had to move the definitions?